book/templates/chapter-template/README.md

```diff
@@ -0,0 +1,138 @@
+
+- **Chapter Folder**: The subfolder where your chapter's content goes should have the same name as the chapter's Landing Page, for example `open-science`.
+
+- **Chapter Content**: Every sub-topic in your chapter should have its own file.
+These should be named by combining the landing page's name with a word that best describes the sub-topic.
+For example, if a sub-topic of the Open Science chapter is about the Infrastructure School of Thought, then its file name could be `open-science-infrastructure.md`
+
+- **Chapter Checklist and Resources**: Like the files that make up the chapter's content, name your chapter's Checklist and Resources files by combining the Landing Page's name with a word that best describes the file.
+This could be `open-science-checklist.md` or `open-science-resources.md` for Checklist and Resources Respectively.
+
+### Add your new files to the book's table of contents
+
+The book-wide table of contents lives in the `_toc.yml` [file](book\website\_toc.yml).
+This file structures _The Turing Way_ and defines the order in which chapters appear.
+Your chapter's files should be added to the `_toc.yml` as appropriate.
+
+To add the chapter on Open Science to Guide 1 in the table of contents, you could do the following:
+
+```
+- file: path/to/Guide 1
+  sections:
+  - title: Open Science
+    file: path/to/open-science
+    sections:
+    - title: Infrastructure School of Thought
+      file: path/to/open-science-infrastructure
+    - title: Checklist
+      file: path/to/open-science-checklist
+    - title: Resources
+      file: path/to/open-science-resources
+```
+
+### Reference external sources appropriately
+
+Ensure external sources are properly referenced and included in _The Turing Way's_ centralised bibtex file as recommended in the [style guide](https://deploy-preview-1459--the-turing-way.netlify.app/community-handbook/style/style-citing.html#ch-style-citing)
+
+### Update the book-wide glossary
+
+_The Turing Way_ maintains a book-wide glossary located in its [Afterword](https://the-turing-way.netlify.app/afterword/glossary.html).
+When writing your chapter, update the book-wide glossary with the key terms in your chapter that readers should remember.
+For each term, cross-reference to the subchapter/section where you explained it.
+
+
+### Cross check your PR
+
+The content of the templates are meant to guide and structure your writing.
+Please remove all of the template's placeholders, tips, and suggestions from your chapter before you submit your PR for review.
+
+### Follow the style and consistency guidelines
+
+As you write your chapter, keep _The Turing Way's_ [style](https://the-turing-way.netlify.app/community-handbook/style.html) and [consistency](https://the-turing-way.netlify.app/community-handbook/consistency.html) recommendations in mind.
+This ensures that your new content is accessible, and fits the overall style, structure, and formatting of _The Turing Way_ book.
```

Comment:
I will move this to the top and start with something like:
Make sure that you have read our [contributors guideline](link), [style guide](link) and [conistency](link) chapter ....